packages/remix-cloudflare-pages/rollup.config.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const path = require("path");
+const babel = require("@rollup/plugin-babel").default;
+const nodeResolve = require("@rollup/plugin-node-resolve").default;
+
+const {
+  buildDir,
+  copyToPlaygrounds,
+  createBanner,
+  getAdapterConfig,
+  getVersion,
+  isBareModuleId,
+} = require("../../rollup.utils");
+
+/** @returns {import("rollup").RollupOptions[]} */
+module.exports = function rollup() {
+  let sourceDir = path.relative(process.cwd(), __dirname) || ".";
+  let outputDir = path.join(
+    buildDir,
+    "node_modules/@remix-run/cloudflare-pages"
+  );
```

Comment:
Shouldn't we make sure Windows won't fail here?

```suggestion
  let outputDir = path.join(
    buildDir,
    "node_modules",
    "@remix-run",
    "cloudflare-pages"
  );
```

Reply:
Yeah.  I noticed a handful of spots in the current config (i.e., https://github.com/remix-run/remix/blob/main/rollup.config.js#L159) that were hardcoding separators so I figured it might not be currently windows friendly and that could be a task in it's own right - but it does seem to be causing issues in CI so I can get those updated :thumbsup: